sympy/solvers/solveset.py

```diff
@@ -737,7 +737,12 @@ def _has_rational_power(expr, symbol):
 
 def _solve_radical(f, symbol, solveset_solver):
     """ Helper function to solve equations with radicals """
-    eq, cov = unrad(f)
+    if unrad(f) is None: # Check if unrad returns NoneType
```

Comment:
The result of `unrad(f)` should be saved in a local variable to avoid having to repeat the call on line 744. Maybe something like this

    res = unrad(f)
    eq, cov = res if res else (f, [])